src/structures/BaseEmoji.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const Emoji = require('./Emoji');
+
+/**
+ * Parent class for `GuildEmoji` and `GuildPreviewEmoji`.
+ * @extends {Emoji}
+ */
+class BaseEmoji extends Emoji {
+  /**
+   * @param {Client} client The instantiating client
+   * @param {Object} data The data for the emoji
+   * @param {Guild} guild The guild the emoji is a part of
+   */
+  constructor(client, data, guild) {
+    super(client, data);
+
+    /**
+     * The guild this emoji is a part of
+     * @type {Guild}
+     */
+    this.guild = guild;
+
+    /**
+     * Whether or not this emoji is animated
+     * @type {boolean}
+     */
+    this.animated = data.animated;
```

Comment:
same as above

Reply:
Thank you for pointing the redundancy out, I didn't notice it, sorry. Anywho, I've pushed the changes, let me know if they're good :+1: